src/relentless/simulate/simulate.py

```diff
@@ -870,3 +882,28 @@ class AnglePotentialTabulator(PotentialTabulator):
 
     def __init__(self, potentials, num):
         super().__init__(potentials=potentials, start=0.0, stop=numpy.pi, num=num)
+
+
+class DihedralPotentialTabulator(PotentialTabulator):
```

Comment:
Not sure if the angle tabulator is also missing this, but it should be noted here that the tabulation runs from -pi to pi, so that someone knows how num relates to the spacing.

Reply:
That's my bad! I did document the range on the angle potential, but missed it on this one.